lms/static/sass/views/_program-details.scss

```diff
@@ -715,6 +709,7 @@ $btn-color-primary: $primary-dark;
     @include float(right);
 
     width: $baseline*15;
+    padding: $baseline*1.5 $baseline*0.5;
```

Comment:
Unless I'm misunderstanding, keeping `padding` and `padding-right` together would mean that originally `padding` would make the left and right padding `$baseline*0.5`, but then `padding-right` would overwrite the right padding to be `$baseline*1.5`. 
If that's the case, my one nitpick is that I would want this padding to be more clear, either by leaving a comment or writing all four params of `padding` in replacement of using `padding-right` underneath.

Reply:
Thanks for the feedback! Was worried about readability, and opted to keep it similar to what was there already, but agree with you, shouldn't need to set padding twice. Have made the appropriate changes :)